stellar_sdk/keypair.py

```diff
@@ -278,12 +278,12 @@ def generate_shamir_mnemonic_phrases(
             message = "shamir_mnemonic must be installed to use method `generate_shamir_mnemonic_phrases`."
             raise ModuleNotFoundError(message) from exc
 
-        secrets = Keypair.random().secret.encode()
```

Comment:
Can you add a `strength` parameter to the function? The default is 128, and it should currently support both 128 and 256.

Reply:
Done and I relaxed it a bit in the code for Shamir as the SLIP says it can be a multiple of 16. I left a comment in the code.

Reply:
Nice 👍